src/main/java/uk/gov/pay/api/validation/CardExpiryValidator.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.pay.api.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.regex.Pattern;
+
+public class CardExpiryValidator implements ConstraintValidator<ValidCardExpiryDate, String> {
+    
+    private Pattern pattern = Pattern.compile("(0[1-9]|1[0-2])\\/\\d{2}");
+    
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        
+        if (value == null) {
+            return false;
+        }
+        
+        return pattern.matcher(value).matches() ? true : false;
```

Comment:
can just be `return pattern.matcher(value).matches();`

Reply:
Done!